src/Attachment.tsx

```diff
@@ -72,19 +112,25 @@ const Video = (props: {
     onLoad?: () => void,
     onClick?: () => void,
 }) => {
-    const src = new URI(props.src);
-    const domain = src.domain();
+    const url = document.createElement('a');
+    url.href = props.src;
+
+    const queryObject = queryStringToObject(url.search);
+    const pathSegments = url.pathname.substr(1).split('/');
```

Comment:
I believe it removes the leading slash character, so that the array from .split does not have an empty first element.